x-pack/test_serverless/functional/test_suites/observability/dataset_quality/dataset_quality_flyout.ts

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { datasetNames, getInitialTestLogs, getLogsForDataset } from './data';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects([
+    'common',
+    'navigationalSearch',
+    'observabilityLogsExplorer',
+    'datasetQuality',
+    'svlCommonNavigation',
+    'svlCommonPage',
+  ]);
+  const testSubjects = getService('testSubjects');
+  const synthtrace = getService('svlLogsSynthtraceClient');
+  const to = '2024-01-01T12:00:00.000Z';
+
+  describe('Dataset quality flyout', () => {
+    before(async () => {
+      await PageObjects.svlCommonPage.login();
```

Comment:
As I pointed already, ideally each test suite (test file) should use minimal required SAML role for authentication and validate customer with this role can access functionality. In case of oblt project there are 2 roles - `viewer` and `editor` + `admin`, that you should use only in case of feature access is restricted. As a reference I use [roles.yml ](https://github.com/elastic/kibana/blob/main/packages/kbn-es/src/serverless_resources/project_roles/oblt/roles.yml)

Reply:
Changed the implementation to use `loginWithRole`. The Dataset Quality plugin requires `admin` role.